### DISCUSSION TITLE
Welcome to 2024-data-short-course Discussions!

### DISCUSSION THREAD

**Discussion by nquarder** [cu-esiil-edu/2024-data-short-course#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Vital-Ahishakiye** [comment]
hello, everyone!

**Reply by nquarder** [reply]
Welcome to the Data Short Course @Vital-Ahishakiye - we are excited to learn with you!

**Comment by WilliamBlacksmith** [comment]
Hello everyone! hope you all are doing well.

**Comment by jarrodredbird** [comment]
hello good morning everyone
